### PR TITLE
Auto close PR test

### PR DESCRIPTION
**Pull request by EVDOG4LIFE**
## What does this PR do?
This PR adds a GitHub workflow to automatically close pull requests and add a comment explaining that the repository is auto-generated and does not accept direct contributions. This is being tested on an unused SDK repository.

## Test Plan
To verify this change:
1. Create a new pull request in the unused SDK repository.
2. Observe that the workflow is triggered automatically.
3. Verify that a comment is added to the pull request with the specified text.
4. Confirm that the pull request is automatically closed.
5. Test reopening the pull request to ensure the workflow triggers again.

## Related PRs and Issues
N/A - This is a standalone change for repository management.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
Yea

### CODE REVIEW

File: .github/workflows/autoclose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Close Pull Request
+        uses: actions/github-script@v7.0.1
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            await github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.name,
+              body: 'This library is auto-generated by Appwrite custom [SDK Generator](https://github.com/appwrite/sdk-generator), and does not accept pull requests directly. To learn more about how you can help us improve this SDK, please check the [contribution guide](https://github.com/appwrite/sdk-generator/blob/master/CONTRIBUTING.md) before sending a pull-request.'
```
Comment (stnguyen90) [inline .github/workflows/autoclose.yml:20]:
```suggestion
              body: 'This library is auto-generated by the Appwrite [SDK Generator](https://github.com/appwrite/sdk-generator), and does not accept pull requests directly. To learn more about how you can help us improve this SDK, please check the [contributing guide](https://github.com/appwrite/sdk-generator/blob/master/CONTRIBUTING.md) before submitting a pull request.'
```

File: .github/workflows/autoclose.yml
Comment (stnguyen90) [inline .github/workflows/autoclose.yml:1]: How do we make sure this only runs for contributors and doesn't run when we need to submit a PR?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Left a comment.

File: .github/workflows/autoclose.yml
Comment (EVDOG4LIFE) [reply .github/workflows/autoclose.yml:1]: I believe this refactor should correct now

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**